services/reddit/reddit-base.js

```diff
@@ -0,0 +1,91 @@
+import Joi from 'joi'
+import { BaseJsonService } from '../index.js'
+
+const tokenSchema = Joi.object({
+  access_token: Joi.string().required(),
+  expires_in: Joi.number(),
+  scope: Joi.string(),
+  token_type: Joi.string(),
+})
```

Comment:
The `expires_in` value is used. The `scope` and `token_type` values are discarded. Suggest

```js
const tokenSchema = Joi.object({
  access_token: Joi.string().required(),
  expires_in: Joi.number().required(),
}).required()
```

here.

Reply:
Good point, fixed!